# STTA 254 (WZ Cas) - WDS - Cas

# Astrométrie et informations générales

In [1]:
import sys
import os
# import du package astrodm
#if 'D:\DOCUMENTS\Astronomie\dev' not in sys.path:
sys.path.insert(0, 'D:\DOCUMENTS\Astronomie\dev')
from astrodm import doublesoutils as do
from IPython.core.display import display, HTML
display(HTML("<style>.container { width:100% !important; }</style>"))
display(HTML("<style>.output_result { width:100% !important; }</style>"))
display(HTML("<style>.prompt { display:none !important; }</style>"))


### Lance une requête au sujet du système mentionné en titre dans le catalogue WDS principal avec les notes si présentent

In [2]:
do.info_src_wds('stta254', '*', True) # True = imprimer les notes WDS.


Il y a 4 résultats pour «STTA254», paire = «*».

Données tirées du Washington Double Stars Catalog (WDS)
Tris sur "Disc" et "Comp"
------------------------------------------------------------------------------------------------------------------------
   WDS       Disc  Comp Obs1 Obs2 pa1 pa2  sep1   sep2   mag1   mag2    DM    Notes n_RAJ2000   RAJ2000     DEJ2000  
                         yr   yr  deg deg arcsec arcsec  mag    mag                             "h:m:s"     "d:m:s"  
---------- ------- ---- ---- ---- --- --- ------ ------ ------ ----- -------- ----- --------- ----------- -----------
00013+6021 STTA254   AB 1874 2019  90  89   58.9  57.80  7.400  8.33 +59 2810     N           00 01 15.85 +60 21 19.0
00013+6021 STTA254   AC 1879 2015 324 324  155.7 155.40  7.400  9.56 +59 2809     N           00 01 15.85 +60 21 19.0
00013+6021 STTA254   AD 1982 2016 118 118  182.0 181.40  7.400 10.35              N           00 01 15.85 +60 21 19.0
00013+6021 STTA254   BD 1879 2015 129 1

## Historique du système

Le système à été découvert par ...

## Instrumentation f/20 en «Lucky Imaging»

L'instrumentation pour le «Lucky Imaging» sur EdgeHD 8 est : ASI 120MM-S + Barlow 2X = f/20 appr. Le champ de la caméra est alors de 160 X 120 arcsec environ.

## Discussion sur les données du catalogue WDS : paire AB
D'après les données de catalogue ci haut, la paire AB avec sep2 = 57.8 arcsec, semble parfaite pour l'instrumentation f/20 et sera observée en premier. À la rigueur, la paire BD (sep2 = 133.70 arcsec) et la paire AC (sep2 = 155.7 arcsec) pouraient tout juste être observées à f/20. Par contre, la paire AD (sep2 = 181.4 arcsec) est trop espacée pour cette instrumentation, mais pourrait éventuellement être mesurée sans la Barlow donc à f/10. Cependant l'exploitation de l'instrumentation à f/10 reste à développer.

### Différences de magnitudes
Les différences de magnitudes sont me semblent acceptables pour l'observation.


## Observations de la paire AB dans le programme «2021-023  - Réglages caméras pour le «Lucky Imaging» (LI)»

# Calibration d'échelle
Dans le programme 2021-023, session 9, j'ai travaillé au développement du flux de travail pour le «Lucky Imaging». Lors de cette session, j'ai effectué le premier essai de la technique de calibration de l'échelle de la caméra en utilisant le masque à fentes de 3 mm et le filtre Ha. Par la suite, j'ai acquis des images (S=1, bloc 1) pour mesurer la paire AB. Je n'ai cependant pas acquis des images de calibration photographiques (darks, etc.).

## Paire AB - résultats d'observation

In [3]:
do.version()
s = do.post_reduction(type_session='complete', ch_prog=os.getcwd())
s.imprimerRapport('c')

astrodm.doublesoutils version 49, avril 2022.


RÉDUCTION COMPLÈTE
----------------------------------------------------------------------------------------------------
id_system     id_WDS     RAJ2000     DEJ2000 id_system_alt1 id_system_alt2 const        remarques
  STTA254 00013+6021 00 01 15.85 +60 21 19.0     "A=WZ Cas"   "rechercher"   Cas "info du WDS ok"
----------------------------------------------------------------------------------------------------
           Système : STTA254
             Paire : AB
         Programme : P2021-023
Nombre de sessions : 3

  CheminProgramme  : D:\DOCUMENTS\Astronomie\dev\med\STTA254\AB\P2021-023
       Fichier log : D:\DOCUMENTS\Astronomie\dev\med\STTA254\AB\P2021-023/post-réduction.log

 ----------------------
           Réduction
 Session   complétée?
 ----------------------
 Session 1 True
 Session 2 True
 Session 3 True


DÉTAILS DES SESSIONS D'OBSERVATIONS
----------------------------------------------------------------------------------

In [4]:
s.moyennes_par_session_df

,S,Date_UTC,epoque_moy,rho_moy,rho_sigma_moy,theta_moy,theta_sigma_moy,deltaTemperatureValide,delaiDepuisAcqEchValide,deltaMatrixReducValide
0,1,2021-10-09,2021.7718,56.4711,0.1786,89.52,0.0424,True,True,True
1,2,2021-10-16,2021.7912,56.4397,0.2163,89.54,0.0424,True,True,True
2,3,2021-10-24,2021.8131,56.4393,0.3518,89.50,0.0600,True,True,True


In [5]:
s.resultats_pour_publication_df

,prog,id_system,pr,con,N,groupe,Dates_UTC,delai_entre_N,Échéance,Sessions,epoque,pa,pa_sig,sep,sep_sig,WDS_o2,WDS_pa2,WDS_sep2,État
0,P2021-023,STTA254,AB,Cas,2,(1),"[2021-10-09, 2021-10-16]",7.0,,"[1, 2]",2021.7815,89.53,0.03,56.4554,0.1402,2019,89,57.8,TOL
1,P2021-023,STTA254,AB,Cas,1,(-),[2021-10-24],NaN,2021-10-31,[3],2021.8131,89.50,0.06,56.4393,0.3518,2019,89,57.8,TAL
